src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCarRentCost = 40;
```

Comment:
Consider using a constant to define the base cost per day of renting a car. This value is a good candidate for a constant because it is a fixed value that does not change and is used in multiple places within the function.